packages/forms/docs/01-overview.md

```diff
@@ -90,6 +90,15 @@ TextInput::make('name')
     ->label(__('fields.name'))
 ```
 
+Optionally, you can have the label automatically translated [using Laravel's localization features](https://laravel.com/docs/localization#introduction) with the `translateLabel()` method:
+
+```php
+use Filament\Forms\Components\TextInput;
+
+TextInput::make('name')
+    ->translateLabel() // Equivalent to `label(__('Name'))`
+```
+
```

Comment:
Please remove this, I didn't include it because I don't think it should exist and its only there for backwards compatibility